R/httr2-translate.R

```diff
@@ -0,0 +1,189 @@
+#' Translate httr2 request to curl command
+#'
+#' Convert an httr2 request object to equivalent curl command line syntax.
+#' This is useful for debugging, sharing requests, or converting to other tools.
+#'
+#' @param .req An httr2 request object created with [request()].
+#' @return A character string containing the curl command.
+#' @export
+#' @examples
+#' \dontrun{
+#' # Basic GET request
+#' request("https://httpbin.org/get") |>
+#'   httr2_translate()
+#'
+#' # POST with JSON body
+#' request("https://httpbin.org/post") |>
+#'   req_body_json(list(name = "value")) |>
+#'   httr2_translate()
+#'
+#' # POST with form data
+#' request("https://httpbin.org/post") |>
+#'   req_body_form(name = "value") |>
+#'   httr2_translate()
+#' }
+httr2_translate <- function(.req) {
```

Comment:
Given that this works with a request object, I think `req_as_curl()` or similar might be a better name.